Resolves #2062 (#2132)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012-ps/dism/Add-AppxProvisionedPackage.md

> You cannot install an app package (.appx) on an operating system that does not support Windows® 8 apps.

Reviewer: ```suggestion
You cannot install an app package (.appx) on an operating system that does not support Windows 8 apps.
```

> Apps are not supported on Server Core installations of Windows Server® 2012, Windows® Preinstallation Environment (Windows PE) 4.0, or on any versions of Windows older than Windows 8 and Windows Server 2012.

Reviewer: ```suggestion
Apps are not supported on Server Core installations of Windows Server 2012, Windows Preinstallation Environment (Windows PE) 4.0, or on any versions of Windows older than Windows 8 and Windows Server 2012.
```

> To install and run apps on Windows Server 2012, you must install the [Desktop Experience](http://go.microsoft.com/fwlink/?LinkId=247330).

Reviewer 2: ```suggestion
To install and run apps on Windows Server 2012, you must install the [Desktop Experience Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn609826(v=ws.11)).
```

> To install and run apps on Windows Server 2012, you must install the [Desktop Experience Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn609826(v=ws.11))..

Reviewer: ```suggestion
To install and run apps on Windows Server 2012, you must install the [Desktop Experience Overview](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn609826(v=ws.11)).
```

Use the *Online* parameter to specify the running operating system on your local computer, or use the *Path* parameter to specify the location of a mounted Windows image.

Use the *PackagePath*, *DependencyPackagePath*, and *LicensePath* parameters to specify the location of all the files that are needed to add the provisioned app package (.appx).
Use these parameters to provision line-of-business apps.

Use the *FolderPath* parameter to specify the location of a folder of unpacked app package (.appx) files that includes any dependency packages and a license file.

To add an app package (.appx) for a particular user, or to test a package while developing your app, use the **Add-AppxPackage** cmdlet instead.

> For more information, including requirements for app package provisioning, see [Sideload Apps with DISM](http://go.microsoft.com/fwlink/?LinkID=231020) and [How to develop an OEM app that uses a custom file](http://go.microsoft.com/fwlink/?LinkID=279989) in the TechNet Library..

Reviewer 2: ```suggestion
For more information, including requirements for app package provisioning, see [Sideload Apps with DISM](https://docs.microsoft.com/windows-hardware/manufacture/desktop/sideload-apps-with-dism-s14) and [How to develop an OEM app that uses a custom file](https://docs.microsoft.com/windows/win32/appxpkg/how-to-develop-oem-app-with-custom-file).
```

> Specifies the maximum output level shown in the logs.
> The default log level is 3.
> The accepted values are as follows:

Reviewer 2: ```suggestion
Specifies the maximum output level shown in the logs. The accepted values are as follows:
```

> - 1 = Errors only
> - 2 = Errors and warnings
> - 3 = Errors, warnings, and information
> - 4 = All of the information listed previously, plus debug output

Reviewer 2: ```suggestion
- Errors
- Warnings
- WarningsInfo (default)
```

Reviewer 2: I think it should be the same as https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2139.

Could you test it in 2012?

Author: Seems like it. Could be a broader issues on the` -LogLevel `parameter in the module... @e0i This could probably be investigated.

Confirmed on 2012, 2012 R2, 2016 and Win10.
